src/PresentationalComponents/ZeroState/AppZeroState.test.js

```diff
@@ -13,27 +12,35 @@ jest.mock(
     })
 );
 
-const intlProviderConfig = {
-    locale: 'en',
-    messages: {}
-};
+describe('AppZeroState component', () => {
 
-describe('NewAppZeroState component', () => {
+    it('renders zero state if there ARE children but NO systems', async () => {
+        render(
+            <MemoryRouter initialEntries={['/some-path']}>
+                <Routes>
+                    <Route path="/some-path" element={
+                        <AppZeroState app="Advisor" customFetchResults={false}>
```

Comment:
*let me show you this one trick* So, since we sorta do have a list/definitions of apps, we could use some `Math.rand` magic to select a random app for the tests.

It would make the tests more chaotic and might help to make the component more robust.

Reply:
this is dope